src/explore/AB_testing/by_date_camps.py

```diff
@@ -151,11 +164,14 @@ def create_df(df, keys): #5/7 Main 'Function for AB pipeline'
     return model_check  
 
 
-
-
 if __name__ == '__main__':
+
+    dataframe = pd.read_csv(DATAFRAME_AB) 
+    #ids = pd.read_csv('/home/allen/RIP_Tensor1/capstone2/Capstone2/src/explore/testing/x1.csv')
+    df_withdates = pd.read_csv(DATAFRAME_WITHDATES) 
+    df_blank = pd.read_csv(DATAFRAME_BLANK) 
     #print(ids.head(), ids.columns)
-    step1 = edit_df()
+    step1 = edit_df(dataframe)
      
```

Comment:
at some point you should be more descriptive with what is happening here.  Here in the workflow and in the function docstrings.